addons/mail/static/src/components/activity/activity.scss

```diff
@@ -2,68 +2,35 @@
 // Layout
 // ------------------------------------------------------------------
 
-.o_Activity {
-    display: flex;
-    flex: 0 0 auto;
-    padding: map-get($spacers, 2);
-}
-
 .o_Activity_detailsUserAvatar {
-    margin-inline-end: map-get($spacers, 2);
-    object-fit: cover;
-    height: 18px;
-    width: 18px;
+    height: 1.4em;
+    width: 1.4em;
+    vertical-align: text-bottom;
+    margin-inline-end: map-get($spacers, 1);
```

Comment:
Why is this one not becoming a `mr-1` for example?

Reply:
For rtl languages since the avatar is rendered as inline text